examples/eiger_use_case.py

```diff
@@ -0,0 +1,23 @@
+'''Virtual datasets: The 'Eiger' use case
+
+https://support.hdfgroup.org/HDF5/docNewFeatures/VDS/HDF5-VDS-requirements-use-cases-2014-12-10.pdf
+'''
+
+import h5py
+import numpy as np
+
+
+files = ['1.h5', '2.h5', '3.h5', '4.h5', '5.h5']
+entry_key = 'data' # where the data is inside of the source files.
+sh = h5py.File(files[0], 'r')[entry_key].shape # get the first ones shape.
+
+layout = h5py.VirtualLayout(shape=(len(files),) + sh, dtype=np.float)
```

Comment:
Isn't this going to produce a 4d layout (as `sh` is already length 3)?  I think this should be `(len(files) * sh[0], ) + sh[1:]`.

Reply:
You're right, done. Thanks!